### DISCUSSION TITLE
MLflow 2.0.0rc0

### DISCUSSION THREAD

**Discussion by harupy** [mlflow/mlflow#7213] [Announcements]
The first release candidate for MLflow 2.0.

## Installation

```bash
# Make sure python version is >=3.8
pip install mlflow==2.0.0rc0
```

## Documentation

[MLflow 2.0.0rc0 documentation](https://output.circle-artifacts.com/output/job/bb07270e-1101-421c-901c-01e72bc7b6df/artifacts/0/docs/build/html/index.html)

<hr /><em>This discussion was created from the release <a href='https://github.com/mlflow/mlflow/releases/tag/v2.0.0rc0'>MLflow 2.0.0rc0</a>.</em>

**Comment by dolfinus** [comment]
What about release notes?

**Reply by BenWilson2** [reply]
They'll be generated with the official 2.0 release :) We're still making some changes and adjustments.

**Comment by cao-nv** [comment]
Do we have the official MLFlow Pipeline?

**Comment by ashishpatel26** [comment]
**What new features and improvements have been released?**
